playground/frontend/lib/modules/editor/components/editor_textarea.dart

```diff
@@ -31,6 +31,15 @@ import 'package:playground/modules/examples/models/example_model.dart';
 import 'package:playground/modules/sdk/models/sdk.dart';
 import 'package:provider/provider.dart';
 
+const kNumberOfStringsToSkip = 16;
+const kJavaRegExp = r'import\s[A-z.0-9]*\;\n\n[(\/\*\*)|(public)|(class)]';
+const kPythonRegExp = r'[^\S\r\n](import|as)[^\S\r\n][A-z]*\n\n';
+const kGoRegExp = r'[^\S\r\n]+\'
+    r'"'
+    r'.*'
+    r'"'
+    r'\n\)\n\n';
+
```

Comment:
I think this is fine as a workaround, but I would like to support examples to have a tag `scroll-to-line: 123` or something like that - so that we're not having to guess how much to scroll programatically. What do you think about this? Is this something you can do?

If not, can you create a JIRA issue to keep this in mind? Thanks

Reply:
fyi @miamihotline

Reply:
Hi @pabloem! Since you've already merged this one, we created a new ticket BEAM-13703 to come back to your suggestion soon.